src/Build/Graph/GraphBuilder.cs

```diff
@@ -48,6 +49,8 @@ internal class GraphBuilder
         private readonly ProjectGraph.ProjectInstanceFactoryFunc _projectInstanceFactory;
         private IReadOnlyDictionary<string, IReadOnlyCollection<string>> _solutionDependencies;
 
+        public bool IsDynamicPlatformNegotiationEnabaled = false;
```

Comment:
```suggestion
        private bool PlatformNegotiationEnabled = false;
```
Does it need to be public?